defaults/main.yml

```diff
@@ -1456,6 +1456,26 @@ owncloud__nginx_access_log_assets: True
 # TODO: Confirm that this variable does what it says.
 owncloud__nginx_client_body_temp_path: ''
 
+owncloud__nginx__php_status: False
+owncloud__nginx__php_status_location:
+  pattern: '~ ^/(php_status|php_ping)$'
+  options: |
+    access_log off;
+    {% if nginx_status_localhost %}
+    {% for address in nginx_status_localhost | unique %}
+    allow {{ address }};
+    {% endfor %}
+    {% endif %}
+    {% if nginx_status %}
+    {% for address in nginx_status | unique %}
+     allow {{ address }};
+    {% endfor %}
+    {% endif %}
+    deny all;
+    include fastcgi_params;
+    fastcgi_index index.php;
+    fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+    fastcgi_pass php_owncloud;
```

Comment:
I see... I have an idea to move the location list in the default nginx server template to role default variables, this way the order of the locations (since it's important) can be controlled better, or default locations could be disabled completely if needed. Not sure yet when that will be ready, but I'll try to look into it soon in this case.